pyproject.toml

```
@@ -41,7 +41,7 @@ doc = [
 home = "https://github.com/pyansys/pyansys-tools-report"
```

These links will be shown in the PyPI page of the package:

```suggestion
Home = "https://reportdocs.pyansys.com/"
Source = "https://github.com/pyansys/pyansys-tools-report"
Tracker = "https://github.com/ansys/pyansys-tools-report/issues"
```

---

Thanks! Adding to PR :)